File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1142.c
#include <stdio.h>

int main(){
    int n;
	scanf("%d", &n);
    int l=1, i=0;
    for(i=0;i<n;i++){
		printf("%d %d %d PUM\n", l, l+1, l+2);
        l+=4;
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1142.cs
using System;

class URI
{
    static void Main() {
        int n = int.Parse(System.Console.ReadLine().Trim());
        int l=1, i;
        for(i=0;i<n;i++){
    		Console.Write("{0} {1} {2} PUM\n", l, l+1, l+2);
            l+=4;
        }
    }
}